app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -598,12 +602,16 @@ def pulp_content_url
         URI(setting(SmartProxy::PULP3_FEATURE, 'content_app_url'))
       end
 
+      def load_balancer_pulp_content_url
+        URI::HTTPS.build(host: registration_url.host, path: pulp_content_url.path)
+      end
```

Comment:
hopefully `setting(SmartProxy::PULP3_FEATURE, 'content_app_url')` doesn't contain any crucial information that would be lost by only using its `path`.

Reply:
@ianballou Would this url be different for a smart proxy that uses an external Pulp? Is that setup supported with load balanced smart proxies?